accounts/backends.py

```diff
@@ -1,26 +1,36 @@
 from django.contrib.auth.models import User
+from django.db.models import Q
 
 
-class EmailAuth(object):
+class CaseInsensitiveAuth(object):
```

Comment:
@yoniLavi are new-style classes not preferred?

Reply:
It's still new style classes, but yes, you're absolutely right; I suppose I've just gotten into the habit of trying to the code I write work both in Python 2 and Python 3, so it's hard for me now that we can actually just use Python 3.